### ISSUE TITLE
explain how people can set up (new) docker containers on disconnected machine

### ISSUE THREAD

**Issue by sebastiaan101** [EHDEN/ARACHNE#41]
add documentation to download and install docker image on machine disconnected from internet

**Comment by konstjar** [comment]
Please use these commands
```
docker save
docker load
```

**Comment by PRijnbeek** [comment]
Yes but we can close it if this is documented as a topic on the wiki.

**Comment by konstjar** [comment]
I think it's out of scope of application, it's straight forward devops task and there is no need to copy/paste documentation from docker.com or here is good answer
https://stackoverflow.com/questions/23935141/how-to-copy-docker-images-from-one-host-to-another-without-using-a-repository

**Comment by PRijnbeek** [comment]
I understand but the documentation should explain how users can install the application. Since our users will have to deal with the offline situation we at least need to give them some guidance. For example we do the same for the installation of Atlas in an off line mode. No need to repeat information elsewhere of course.
